src/pages/LiveMatchDiary/NewDiary.js

```
@@ -57,36 +71,34 @@ const NewDiary = () => {
         e.preventDefault();
 
-        const diaryData = {
-            id: isEditing ? location.state.id : Date.now(),
-            team,
+        const postData = {
             title,
             content,
-            date: isEditing ? location.state.date : new Date().toISOString().slice(0, 10).replace(/-/g, '.'),
-            image: image ? image : null,
-            teamLogo: teamLogos[team] || null
+            image: image || null,
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)